### ISSUE TITLE
foutmelding: kan resultaten niet laden (error: coult not load results)

### ISSUE THREAD

**Issue by leijdenf** [dropcart/dropcart#117]
When I get into "beheer" and get for example customers of articles etc I get an error; coult not load results.  Please help were to look

**Comment by egogo-nl** [comment]
This error shows up when the JSON request is not loaded as expected. Please send us the header & result of the JSON request when the page is loaded. You can see this result in the console of your browser.

**Comment by leijdenf** [comment]
Hello,

 

I don’t know what your meaning.

Can you please send me some details where to find it

 

Is this what you are meaning??

 



 

Met vriendelijke groet,

 

Frank van Leijden

 

Van: Egogo Websolutions [mailto:notifications@github.com] 
Verzonden: dinsdag 12 april 2016 14:45
Aan: dropcart/dropcart <dropcart@noreply.github.com>
CC: leijdenf <f.van.leijden@vanleijden.com>
Onderwerp: Re: [dropcart/dropcart] foutmelding: kan resultaten niet laden (error: coult not load results) (#117)

 

This error shows up when the JSON request is not loaded as expected. Please send us the header & result of the JSON request when the page is loaded. You can see this result in the console of your browser.

**Comment by egogo-nl** [comment]
For information how to use the console (in Chrome) please view this page: https://developer.chrome.com/devtools/docs/console

In short, press `Control` + `Shift`  + `J` before loading the page. The JSON request should now be visible in the Console tab.

**Comment by leijdenf** [comment]
Is dit wat je zocht, Is this what you are looking for

 

These are from de backend to some menu’s

 



 



 

And these from the frontside when I go to a menu

 



 

 

 

Met vriendelijke groet,

 

Frank van Leijden

 

Van: Egogo Websolutions [mailto:notifications@github.com] 
Verzonden: dinsdag 12 april 2016 16:40
Aan: dropcart/dropcart <dropcart@noreply.github.com>
CC: leijdenf <f.van.leijden@vanleijden.com>
Onderwerp: Re: [dropcart/dropcart] foutmelding: kan resultaten niet laden (error: coult not load results) (#117)

 

For information how to use the console (in Chrome) please view this page: https://developer.chrome.com/devtools/docs/console

In short, press Control + Shift + J before loading the page. The JSON request should now be visible in the Console tab.

**Comment by egogo-nl** [comment]
Unfortunately I can't see any attachments. Please add your comment through the Github website and attach your files instead of replying by mail.

**Comment by leijdenf** [comment]
Is this what you are looking for
These are from de backend to some menu’s

GET http://tonershopnederland.nl/favicon.ico 500 (Internal Server Error)
jquery.min.js:4 

GET http://tonershopnederland.nl/beheer/includes/json/dc_order_list.json?1undefined&show=new&sort_column=orderId&sort_order=desc&offset=0&limit=25 500 (Internal Server Error)

And these from the frontside when I go to a menu
favicon.ico:1 GET http://tonershopnederland.nl/favicon.ico 500 (Internal Server Error)

**Comment by egogo-nl** [comment]
It looks like the .htaccess file doesn't handle .json files correctly. Could you try the following:
Change in this line https://github.com/dropcart/dropcart/blob/master/beheer/dc_customer_admin.php#L30 `dc_customer_list.json` to `dc_customer_list.php`.

If this works and you get a result, you can also do this with other files which are not loading correctly.

**Comment by leijdenf** [comment]
It looks that this works. I just changed 3 files and all 3 works now. Is there another solution then change everytime a file because at frontside at the website i have also problems with subpages

**Comment by leijdenf** [comment]
The problem is fixed. I choosed another provider. Moved the domain to this provider, uploades all the content and everything works, without changing any files.
One question before you close this issue : Is there somewhere an tutorial for working with dropcart?

**Comment by egogo-nl** [comment]
Good the hear it is now working. We do not yet have a documentation available, but you can check the installation instructions here: https://github.com/dropcart/dropcart/blob/master/README.md.
